src/index.js

```diff
@@ -5,15 +5,24 @@ import makeModifiers from './makers/makeModifiers'
 import makeOriginalClass from './makers/makeOriginalClass'
 import makeStates from './makers/makeStates'
 
+const DEFAULT_OPTIONS = {
+  separators: ['__', '--']
```

Comment:
wdyt if we name these separators?
e.g.
```js
separators: { beforeElement: '__', beforeModifier: '--' }
```

then it would be more easy to read code:

```js
const { beforeElement } = options.separators
const blockName = block + (element ? `${beforeElement}${element}` : '')
```